tests/acceptance/features/lib/TextEditorPage.php

```diff
@@ -98,8 +98,7 @@ public function createTextFile(
 		$name = null,
 		$useDefaultFileExtension = false
 	) {
-		$newFileFolderButton
-			= $this->find("xpath", $this->newFileFolderButtonXpath);
+		$newFileFolderButton = $this->filesPageCRUDFunctions->findNewFileFolderButton();
 
 		if ($newFileFolderButton === null) {
```

Comment:
``findNewFileFolderButton()`` will already have thrown an exception if this is null
``if`` block can be deleted